src/components/views/beta/BetaCard.tsx

```diff
@@ -36,17 +36,27 @@ interface IProps {
     featureId: string;
 }
 
-export const BetaPill = ({ onClick }: { onClick?: () => void }) => {
+interface IBetaPillProps {
+    onClick?: () => void;
+    tooltipTitle?: string;
+    tooltipCaption?: string;
+}
+
+export const BetaPill = ({
+    onClick,
+    tooltipTitle = _t("This is a beta feature"),
+    tooltipCaption = _t("Click for more info"),
+}: IBetaPillProps) => {
     if (onClick) {
         return <AccessibleTooltipButton
             className="mx_BetaCard_betaPill"
-            title={_t("This is a beta feature. Click for more info")}
+            title={`${tooltipTitle} ${tooltipCaption}`}
```

Comment:
Would you please consider to move the whitespace character to `"This is a beta feature"` above for Chinese/Japanese users.

 `"This is a beta feature. "` would be appreciated.